pkg/nss/runner_windows.go

```diff
@@ -5,19 +5,45 @@ package nss
 
 import (
 	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
 )
 
 // CertUtilRunner encapsulates certutil commands.
-type CertUtilRunner struct{}
+type CertUtilRunner struct {
+	binaryPath string
+}
 
 // NewCertUtilRunner returns a new CertUtilRunner.
 // It returns an error if the certutil binary is not found.
 func NewCertUtilRunner() (*CertUtilRunner, error) {
-	return nil, fmt.Errorf("not implemented")
+	ex, err := os.Executable()
+	if err != nil {
+		return nil, fmt.Errorf("getting executable path: %w", err)
+	}
+
+	dir := filepath.Dir(ex)
+	bin := filepath.Join(dir, "mar-tools", "certutil.exe")
+
+	_, err = os.Stat(bin)
+	if err != nil {
+		return nil, fmt.Errorf("failed to find certutil binary: %w", err)
+	}
+
+	return &CertUtilRunner{binaryPath: bin}, nil
+
 }
 
 // Run runs the certutil command with the given arguments.
-// It returns the combined output of stdout and stderr.
+// It returns the combined output of stdout and stderr in the error, if any.
 func (r *CertUtilRunner) Run(args ...string) error {
-	return fmt.Errorf("not implemented")
+	cmd := exec.Command(r.binaryPath, args...) // #nosec G204
+
+	out, err := cmd.CombinedOutput()
+	if err != nil {
+		return fmt.Errorf("%s: %w", string(out), err)
+	}
+
+	return nil
```

Comment:
duplicated code with https://github.com/massalabs/station/blob/e88e04f40afee7af81b101a384d8b43064c5ae7c/pkg/nss/runner_unix.go#L28

Reply:
Indeed, but since those functions are for 2 different structures, I don't see how we can fix this 
Do you have an idea ?

Reply:
maybe by moving the struct declaration into runner.go, move also that function, keep NewCertUtilRunner in OS specific go file.

Reply:
@gregLibert does this remark block from merging? because otherwise LGTM